pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[project]
+name = "linkml_nmr_scicat_ingesting"
+version = "0.0.1"
+authors = [
+  { name="Runbo Jiang", email="rjiang2@lbl.gov" },
+]
+description = "A package to ingest nmr data to SciCat"
+readme = "README.md"
+requires-python = ">=3.8"
```

Comment:
The [current recommendation](https://scientific-python.org/specs/spec-0000/) for how far back to support would have us requiring python 3.10 or above. Let's go with that.